doc/md/integrations/github-actions.mdx

```diff
@@ -379,4 +380,33 @@ Configure this action with the following inputs:
 
 #### `cloud-token`
 
-**Required** The Atlas Cloud token to use for authentication.
\ No newline at end of file
+**Required** The Atlas Cloud token to use for authentication.
+
+## `ariga/setup-atlas`
+
+The [ariga/setup-atlas](https://github.com/ariga/setup-atlas) action can be used to install Atlas from a github action.
```

Comment:
```suggestion
The [ariga/setup-atlas](https://github.com/ariga/setup-atlas) action can be used to install the Atlas CLI from a GitHub Actions workflow.
```

Reply:
In the previous PR I used `Atlas CLI` and Mashraki suggested to use just `Atlas`

Let's keep it consistent? which one should I use?

Reply:
`Atlas` , sorry.

Reply:
👍 
Done